src/creature_states_scavn.c

```diff
@@ -211,7 +209,10 @@ short creature_scavenged_disappear(struct Thing *thing)
 
 short creature_scavenged_reappear(struct Thing *thing)
 {
-    return _DK_creature_scavenged_reappear(thing);
+    struct CreatureControl* cctrl = creature_control_get_from_thing(thing);
+    create_effect(&thing->mappos, get_scavenge_effect_element(cctrl->byte_9C), thing->owner);
```

Comment:
I get that this is not something you introduced, but it looks like get_scavenge_effect_element gets an 'effect', not an 'effect element' right?